test/widgets/drive_detail_page_test.dart

```diff
@@ -255,13 +281,13 @@ void main() {
       });
     });
 
-    testWidgets('Can navigate to drive chat page', skip: true, (WidgetTester tester) async {
+    testWidgets('Can navigate to drive chat page', (WidgetTester tester) async {
       await pumpMaterial(tester, DriveDetailPage.fromDrive(drive));
       await tester.pump();
 
       await tester.tap(find.byKey(const Key('driveChatButton')));
-      await tester.pumpAndSettle();
-
+      await tester.pump();
+      await tester.pump();
```

Comment:
multiple pumps? why is that needed?

Reply:
because of the stream

Reply:
Good to know! But `pumpAndSettle` does not work?